sharepoint/sharepoint-ps/sharepoint-online/Start-SPOCopilotAgentInsightsReport.md

```diff
@@ -0,0 +1,99 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/start-spocopilotagentinsightsreport
+applicable: SharePoint Online
+title: Start-SPOCopilotAgentInsightsReport
+schema: 2.0.0
+author: bhagatshweta
+ms.author: bhagatshweta
+ms.reviewer:
+manager: hikakar
+---
+ 
+# Start-SPOCopilotAgentInsightsReport
+
+## SYNOPSIS
+
+This cmdlet enables administrator to trigger the build of a new Copilot Agent Insight report for the last N days.
```

Comment:
>for the last N days [](http://example.com/codeflow?start=93&length=19)

"for the given number of days" or something better -- ie we can do it without "N"

Reply:
applies to any place you're saying "N"

Reply:
updated at all places and removed "N".